kong/plugins/azure-functions/handler.lua

```diff
@@ -119,6 +120,16 @@ function azure:access(config)
   local response_status = res.status
   local response_content = res:read_body()
 
+  if var.http2 then
+    response_headers["Connection"] = ""
```

Comment:
is there a reason to set them to an empty string instead of removing them all together by setting them to `nil`?

Reply:
Gave the same feedback in person to @rainest - along with a couple of other things.